\documentclass[11pt,letterpaper]{article}
\usepackage[margin=1in]{geometry}
\usepackage[]{natbib}
\bibpunct[; ]{[}{]}{,}{n}{}{;} 
\bibliographystyle{unsrtnat}
\setcitestyle{plain}
\begin{document}

\large{\noindent {\bf Comparative population genomics of maize domestication and improvement.} 
M.B. Hufford, X. Xun, J. van Heerwaarden, T. Pyh\"aj\"arvi, J-M. Chia, R.A. Cartwright, R.J. Elshire, J.C. Glaubitz, K.E. Guill, S. Kaeppler, J. Lai, P.L. Morrell, L.M. Shannon, C. Song, N.M. Springer, R.A. Swanson-Wagner, P. Tiffin, J. Wang , G. Zhang, J. Doebley, M.D. McMullen, D. Ware, E.S. Buckler, S. Yang, J. Ross-Ibarra. 
\textsc{Nature Genetics}, 44(7):808â€“811, 2012.}

\vspace{1cm}

\noindent Early crossing studies between maize and teosinte led Nobel laureate George Beadle to assert that as few as 5 loci could be responsible for the dramatic morphological changes associated with maize domestication \citep{beadle1972mystery}, a conclusion apparently supported by decades of subsequent genetic mapping \citep{briggs2007linkage}.
Initial population genetic investigations of a limited number of loci, however, began to hint that other genes may have been targeted by selection \citep{tenaillon2004selection, Wright2005}.
This paper reports the first whole-genome analysis of maize domestication, expanding signficantly on the results of earlier selection studies. 
We found evidence for hundreds of loci targeted by selection during domestication, and patterns of haplotype differentiation suggest the majority were subjected to stronger selection than the large-effect loci identified in previous mapping efforts.
This result highlights the complexity of maize domestication, suggesting domestication has involved selection on a number of traits beyond the obvious morphological changes.
The high resolution of our selection analysis enabled us to identify individual targets of selection, revealing that a substantial proportion were noncoding regions absent of genes.
Our microarray evaluation of transcriptome-wide gene expression in both maize and its wild ancestor extended these results, finding that selection during domestication targeted cis-regulatory variation that changed expression patterns of selected loci.\\

\noindent Finally, this paper provided substantial evidence for the importance of diversity in crop wild relatives, documenting both the importance of standing genetic variation --- as opposed to selection on new mutations --- as well as the loss of functionally relevant diversity at genes linked to targets of selection.\\

\noindent The paper has been cited more than 180 times, and our population genetic approaches have since been broadly applied by other researchers to a number of other crops species \citep[e.g.][]{qi2013genomic,jordan2015haplotype,zhou2015resequencing}.


%\subsection*{Revising evolutionary models of crop domestication}
%Decades of traditional mapping approaches argued that a handful of loci could effect much of the evolutionary change during domestication, and classical models of selective sweeps  suggested that such loci should result from strong selection on new mutations. 
%Our population genomic analysis of maize and teosinte genomes  \citep{hufford2012comparative} identified hundreds of loci targeted by selection during maize domestication, and suggested that selection on new mutations is likely quite rare.
%In fact, our detailed looks at individual loci of large effect cloned by traditional mapping methods show that even at these loci selection evolution has occurred via selection on standing genetic variation on multiple beneficial mutations \citep{wills2013many} or standing genetic variation \citep{wills2013many, studer2011identification}.
%
%\subsection*{Unique sources of adaptive variation}
%Protein-coding genes have often been considered the major source of functional variation, and evidence from small model genomes such as \emph{Arabidopsis} \citep[e.g.][]{hancock2011adaptation} supports such claims. 
%Like maize, most flowering plants have large genomes (median 1.8Gb), and our work has highlighted that other forms of variation likely play a much more important role in such large genomes.  
%We have shown that selection has taken extensive advantage of regulatory \citep{swanson2012reshaping,pyhajarvi2013complex} and structural variation --- including inversions \citep{pyhajarvi2013complex,fang2012megabase}, transposable element insertions \citep{studer2011identification,makarevitch2015transposable}, and likely copy-number variation \citep{chia2012maize} --- in both natural and domesticated populations. Such noncoding variation is likely responsible for a substantial fraction of adaptive variation in most plant genomes.


\bibliography{jri.bib}
\end{document}
